ludwig/features/image_feature.py

```diff
@@ -224,6 +235,33 @@ def _read_image_if_bytes_obj_and_resize(
                 "#image-features-preprocessing".format([img_height, img_width, num_channels], img.shape)
             )
 
+        # casting and rescaling
+        img = img.type(torch.float32) / 255
+
+        if standardize_image == "imagenet1k":
```

Comment:
nit: could we store this as a constant at the top of the file?

Reply:
added IMAGENET1K constant